StepDefinitions/LoginStep.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using SpecFlowProject_QA_MARS;
+using SpecFlowProject_QA_MARS.Pages;
+using SpecFlowProject_QA_MARS.Utils;
+
+namespace SpecFlowProject_QA_MARS.StepDefinition
+{ 
+    [Binding]  
+    public class LoginStep : Start
+    {
+
+        SignIn SignInObj = new SignIn();
```

Comment:
Please make sure you use constructor it! Please fix where you think makes sense to use constructor!